src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -71,39 +77,46 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         switch (column.key) {
             case 'properties':
                 return (
-                    <IconButton
-                        iconProps={{ iconName: 'EntryView' }}
-                        title={t('advancedSearch.inspectProperty')}
-                        ariaLabel={t('advancedSearch.inspectProperty')}
-                        onClick={(event) => {
-                            event.stopPropagation();
-                        }}
-                        className={'cb-scenes-action-button'}
+                    <PropertyInspectorCalloutButton
+                        twinId={`${currentTwin?.$dtId}`}
+                        adapter={adapter}
                     />
                 );
             default:
                 return <span>{fieldContent}</span>;
         }
     };
+    const selection = new Selection({
+        getKey(item, index?) {
+            return item + index;
+        },
+        onSelectionChanged: () => {
+            setTwin(selection.getSelection());
+            onTwinSelection(currentTwin);
```

Comment:
this is undefined in your stories which throws a console error when you select a row. Can you please pass the callback to prevent this?